dialogue/TownNPCs/Olive/RandomCardinalAzuraBanana/CardinalAzuraBanana_Olive_Random1.json

```diff
@@ -11,11 +11,17 @@
       "I'd often have to stop her from going out of town, as our parents told us the forest isn't safe...",
       "But if she can handle it, I definitely could as well, right?"
     ],
+    "pl": [
+      "W przeszłości, dawno zanim <c=pistachio>moja młodsza siostra</c> wyszła z Cornucopii szukając przygód, często bawiłyśmy się około miasteczka.",
+      "Często musiałam przewstrzymać ją od uciekania od miasteczka, ponieważ nasze rodzice mowiły nam, że las jest grożny...",
+      "Ale jeśli ona daje sobie radę, to ja na pewno też mogłabym, tak?"
```

Comment:
"Ale jeśli ona daje sobie radę, to ja na pewno też bym dała, prawda?" a few small changes